internal/commands/server/plan_list.go

```diff
@@ -18,6 +22,17 @@ func PlanListCommand() commands.Command {
 
 type planListCommand struct {
 	*commands.BaseCommand
+	showCost bool
+	duration time.Duration
+}
+
+// InitCommand initializes the command flags
+func (s *planListCommand) InitCommand() {
+	flagSet := &pflag.FlagSet{}
+	flagSet.BoolVar(&s.showCost, "cost", false, "Show cost information for plans")
+	flagSet.DurationVar(&s.duration, "duration", time.Hour, "Duration for cost calculation (e.g., 1h, 24h, 168h for week, 720h for month)")
```

Comment:
Would be nice to support `hourly` and `monthly` as the values here in addition to duration strings, to have same options as we display in GUI. In hub and in our other materials we highlight the monthly cost so could have that as the default value here as well. Also could add pricing as prefix to the argument name to make it bit more obvious, e.g., `--pricing-duration=hourly`